src/main/java/callhub/connect/controllers/FileController.java

```diff
@@ -103,4 +103,28 @@ public ResponseEntity<Object> addPDFToSession(@RequestParam("file") MultipartFil
             return new ResponseEntity<>(e.getMessage(), headers, HttpStatus.BAD_REQUEST);
         }
     }
+
+    @PutMapping("/update/{id}")
+    public ResponseEntity<String> updateFile(@PathVariable String id, @RequestParam("file") MultipartFile newFile) {
```

Comment:
The parameters accepts a `newFile` however the argument is formally `file` in postman, please make this consistent otherwise it might cause confusion when making the request!